### PR TITLE
Update Session Limits documentation #227

### PR DESCRIPTION
**Pull request by hnjolles1**
Update Session Limits documentation #227
- session limits
- terminology
- guides

### CODE REVIEW

File: docs/video-miners/terminology.md [hunk 1]
```diff
@@ -3,55 +3,66 @@ title: Terminology
...
+
+## Capacity
+
+Capacity refers to the total transcoding that an orchestrator or transcoder can provide in an acceptable time frame (i.e. real-time). At the moment, orchestrators and transcoders can set their capacities based on the maximum number of sessions that they can support. For an orchestrator, this maximum session count refers to the maximum number of simultaneous jobs they can receive from broadcasters. For a transcoder, this maximum session count refers to the maximum number of simultaneous transcoding tasks they can receive from broadcasters.
 
-Additional details on transcoding can be found in
-[this blog post](https://livepeer.com/blog/intro-to-transcoding).
+## Concurrent Sessions
+
+***See*** the definition of [Sessions](video-miners/terminology#session)
```
Comment (hthillman) [inline docs/video-miners/terminology.md:18]: `See` is capitalized here but below it's not, should we stay consistent?

File: docs/video-miners/terminology.md [hunk 2]
```diff
@@ -3,55 +3,66 @@ title: Terminology
...
-  that the broadcaster obtains by authenticating with the orchestrator. This
-  session usually tracks streams closely, but does not necessarily map 1:1 to
-  streams - there are cases where there may be multiple sessions used for the
-  same stream
-- A session on a transcoder refers to an individual transcoding task (for a
-  segment) that is sent by the orchestrator
+## Video Miners
 
-## Capacity
+***see*** [Roles and Responsibilities](/video-miners/)
```
Comment (hthillman) [inline docs/video-miners/terminology.md:64]: here

File: docs/video-miners/terminology.md [hunk 3]
```diff
@@ -3,55 +3,66 @@ title: Terminology
...
 
-Capacity refers to the total transcoding that an orchestrator or transcoder can
-provide in an acceptable time frame (i.e. real-time). At the moment,
-orchestrators and transcoders can set their capacities based on the maximum
-number of sessions that they can support. For an orchestrator, this maximum
-session count refers to the maximum number of simultaneous jobs they can receive
-from broadcasters. For a transcoder, this maximum session count refers to the
-maximum number of simultaneous transcoding tasks they can receive from
-broadcasters.
+***see*** [About Video Mining](/video-miners/)
```
Comment (hthillman) [inline docs/video-miners/terminology.md:68]: here

**Review by hthillman — CHANGES_REQUESTED** [review]
left one extremely minor comment. Overall, this looks great!

File: docs/video-miners/terminology.md (hunk 1 above)
Comment (hnjolles1) [reply docs/video-miners/terminology.md:18]: yes, it should be capitalized.

File: docs/video-miners/terminology.md (hunk 2 above)
Comment (hnjolles1) [reply docs/video-miners/terminology.md]: done

File: docs/video-miners/terminology.md (hunk 3 above)
Comment (hnjolles1) [reply docs/video-miners/terminology.md]: done

**Review by hthillman — APPROVED** [review]

**Pull request merged**